x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_tables/shared_columns_tests.tsx

```diff
@@ -12,8 +12,10 @@ import {
 } from '../../../../../__mocks__';
 import '../../../../__mocks__/engine_logic.mock';
 
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Ahh shoot, I think I see why this file was flagged. It kinda shouldn't be though, because is a test file (or rather a test helper file - it has a command that's run in 2 separate test files), but I didn't want it auto-running on the `jest` command. Is there any way to convert it to a `.test.tsx` file without Jest complaining that it doesn't have any valid tests?

Reply:
There are a number of directories that are "safe" to place these types of files in (such as `__fixtures__` or `test_helpers`). Do any of these sound like a workable location for you? The directory can be placed anywhere within your plugin, it's not limited to a specific location:

https://github.com/elastic/kibana/blob/be9f68ad31c15a8430d88414708925051d1a0910/.eslintrc.js#L108-L131

Reply:
Ahh perfect!! Yes, putting it in a nested `test_helpers/` folder would be amazing. Do you want me to make that fix in a separate PR Larry or do you prefer to make that change in here?

Reply:
I can move it as part of this PR if that'd be easier. I'm flexible though

Reply:
That would awesome if you could. Thanks a billion Larry! 🙇

Reply:
Done in [`b359414` (#93997)](https://github.com/elastic/kibana/pull/93997/commits/b3594146c7298db28d16fe0b356aa2224af0eeeb)

Reply:
Looks terrific, thank you so much!